js/index.js

```diff
@@ -17,67 +17,107 @@ let state = {
   greenPeppers: true,
   whiteSauce: false,
   glutenFreeCrust: false
-};
+}
 
 // This function takes care of rendering the pizza based on the state
 // This function is triggered once at the beginning and every time the state is changed
 function renderEverything() {
-  renderPepperoni();
-  renderMushrooms();
-  renderGreenPeppers();
-  renderWhiteSauce();
-  renderGlutenFreeCrust();
-
-  renderButtons();
-  renderPrice();
+  renderPepperoni()
+  renderMushrooms()
+  renderGreenPeppers()
+  renderWhiteSauce()
+  renderGlutenFreeCrust()
+  renderButtons()
+  renderPrice()
 }
 
 function renderPepperoni() {
   document.querySelectorAll('.pep').forEach(onePep => {
     if (state.pepperoni) {
-      onePep.style.visibility = 'visible';
+      onePep.style.visibility = 'visible'
     } else {
-      onePep.style.visibility = 'hidden';
+      onePep.style.visibility = 'hidden'
     }
-  });
+  })
 }
 
 function renderMushrooms() {
-  // Iteration 1: set the visibility of `<section class="mushroom">`
+  document.querySelectorAll('.mushroom').forEach(mushroom => {
+    mushroom.style.visibility = state.mushrooms ? 'visible' : 'hidden'
```

Comment:
Quiero creer que es aca donde quisiste usar if else, si es asi, no puedes solo usarlo, el detalle es que en la asignación si colocas if-else esta estructura no retorna nada, solo ejecuta. tendrias que usar return dentro de los bloques, lo cual incluso se ve sucio. Asi que tu solución es la correcta.

Asi se ve con if else
```suggestion
    mushroom.style.visibility = if(state.mushrom) { return ''visible }else{ return 'hidden' }
```